docs/4.2/quickstart.md

```diff
@@ -98,24 +98,31 @@ Congratulations - you are now running Teleport!
 ## Step 3: Create a User Signup Token
 
 We've got Teleport running but there are no users recognized by Teleport Auth
-yet. Let's create one for your OS user. In this example the OS user is
-`teleport` and the hostname of the node is `grav-00` .
+yet. Let's create one for your OS user. 
+
+For the following examples: `teleport@grav-00`, the **OS user** is
+`teleport`, and the **hostname** of the node is `grav-00`.
+
+Replace `OS user` and `hostname` as it fits your OS: `<your OS user>@<your hostname>`
 
 !!! info "OS User Mappings"
 
-    The OS user `teleport` must exist! On Linux, if it
-    does not already exist, create it with `adduser teleport`. If you do not have
+    If you intend to use `teleport` as OS user (as shown in our examples), OS user `teleport` must exist! 
+    And you must explicitly state user as `teleport` in some subsequent commands. (We will remind you).
+    
+    On Linux, if it does not already exist, create it with `adduser teleport`. If you do not have
     the permission to create new users on the VM, run `tctl users add teleport
-    <your-username> ` to explicitly map ` teleport` to an existing OS user. If you
-    do not map to a real OS user you will get authentication errors later on in
+    <your-username> ` to explicitly map ` teleport` to an existing OS user. 
+    
+    If you do not map to a real OS user you will get authentication errors later on in
     this tutorial!
 
 ``` bash
 # A new Teleport user will be assigned a
 # mapping to an OS user of the same name
 # This is the same as running `tctl users add teleport teleport`
 [teleport@grav-00 ~]$ tctl users add teleport
-Signup token has been created and is valid for 1 hours. Share this URL with the user:
+User teleport has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h0m0s:
```

Comment:
It's not a huge deal, but as a rule we should add these updates to a new '5.0' docs, since this update won't be live until then. ( I think it's ok for now, since we're a scrappy team working on docs )

Reply:
ah that makes sense, I will take that into account next time :)